### PR TITLE
Refactor constants

### PR DESCRIPTION
**Pull request by MihhailSokolov**
## Description
This PR creates a new class in `core/data` called `Constants` which contains several constants used throughout the project.

## Motivation and context
It's a good practice to keep the constants in one place instead of hard-coding them in several places.

## Testing
Regression testing - ran existing tests to make sure everything works as it was.

## Task list  
- [x] Create `Constants` class and put all hard-coded constants there

### CODE REVIEW

File: core/src/main/java/eu/fasten/core/data/Constants.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package eu.fasten.core.data;
+
+public class Constants {
+
+    public static final String mvnCoordinatePartsJoin = ":";
```
Comment (ilyagrishkov) [inline core/src/main/java/eu/fasten/core/data/Constants.java:5]: I would call it mvnCoordinateSeparator

File: core/src/main/java/eu/fasten/core/data/Constants.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    public static final String mavenForge = "mvn";
```
Comment (ilyagrishkov) [inline core/src/main/java/eu/fasten/core/data/Constants.java:7]: You use `mvn` in other cases, but `maven` here. I think this constant should be called `mvnForge`

File: core/src/main/java/eu/fasten/core/data/Constants.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    public static final String opalGenerator = "OPAL";
+
+    public static final int transactionRestartLimit = 3;
+
+    public static final int insertionBatchSize = 4096;
+
+    public static final String mvnRepoEnvVariable = "MVN_REPO";
```
Comment (ilyagrishkov) [inline core/src/main/java/eu/fasten/core/data/Constants.java:15]: Maybe also add FASTEN_DB_PASS env variable constant?

**Review by ilyagrishkov — CHANGES_REQUESTED** [review]

File: core/src/main/java/eu/fasten/core/data/Constants.java (hunk 1 above)
Comment (MihhailSokolov) [reply core/src/main/java/eu/fasten/core/data/Constants.java:5]: Renamed

File: core/src/main/java/eu/fasten/core/data/Constants.java (hunk 2 above)
Comment (MihhailSokolov) [reply core/src/main/java/eu/fasten/core/data/Constants.java:7]: Renamed it as well

File: core/src/main/java/eu/fasten/core/data/Constants.java (hunk 3 above)
Comment (MihhailSokolov) [reply core/src/main/java/eu/fasten/core/data/Constants.java:15]: Sure, added it along with `PGPASSWORD`

**Review by ilyagrishkov — APPROVED** [review]
LGTM

**Pull request merged**